pkg/ast/pipesearch/searchHandler.go

```diff
@@ -55,19 +55,39 @@ Example incomingBody
 
 finalSize = size + from
 */
-func ParseSearchBody(jsonSource map[string]interface{}, nowTs uint64) (string, uint64, uint64, uint64, string, int) {
+func ParseSearchBody(jsonSource map[string]interface{}, nowTs uint64) (string, uint64, uint64, uint64, string, int, error) {
 	var searchText, indexName string
 	var startEpoch, endEpoch, finalSize uint64
 	var scrollFrom int
+	var errorMessage string
+
+	knownFields := map[string]bool{
+		"searchText": true,
+		"indexName":  true,
+		"startEpoch": true,
+		"endEpoch":   true,
+		"size":       true,
+		"scroll":     true,
+		"from":       true,
+	}
```

Comment:
This is not correct. Some of these fields are optional . This is probably why the end 2 end tests might be failing